### PR TITLE
refactor(Card): convert to hooks

### LINKED ISSUE #110: Convert <Card /> to hooks

### PR DESCRIPTION
**Pull request by HorusGoul**
Closes #110

### CODE REVIEW

File: src/components/shared/card/Card.tsx [hunk 1]
```diff
@@ -1,20 +1,14 @@
 import classNames from "classnames";
 import * as React from "react";
-
 import "./Card.scss";
 
-interface ICardProps extends React.Props<any> {
+interface CardProps {
   className?: string;
+  children: React.ReactNode;
 }
```
Comment (marcosvega91) [inline src/components/shared/card/Card.tsx:8]: Hellooo,

In a previous PR I have changed the interface removing the prefix I too as you have done but as I see it is a naming convention used by repo owners so I think that is better to leave as is. What you think?

```suggestion
interface ICardProps {
  className?: string;
  children: React.ReactNode;
}
```

**Review by marcosvega91 — CHANGES_REQUESTED** [review]
thanksss I have left only one comment

File: src/components/shared/card/Card.tsx (hunk 1 above)
Comment (HorusGoul) [reply src/components/shared/card/Card.tsx:8]: As the repo owner, I can say that it's better without the `I` prefix, but don't focus too much on that right now, as it's something that can be easily replaced later. 

Thank you!

Comment (marcosvega91) [reply src/components/shared/card/Card.tsx:8]: Hahahha Sorry I didn't see that you were the repo owner

Comment (HorusGoul) [reply src/components/shared/card/Card.tsx:8]: No problem xD :smile:

**Review by mpeyper — APPROVED** [review]

**Pull request merged**